collectibles/collectibles.py

```diff
@@ -0,0 +1,223 @@
+# Copyright (c) 2021 - Jojo#7791
+# Licensed under MIT
+
+import logging
+from datetime import datetime
+from typing import Literal, Optional, Union
+
+import discord
+from jojo_utils import Menu, positive_int
+from redbot.core import Config, bank, commands
+from redbot.core.bot import Red
+from redbot.core.utils.chat_formatting import pagify
+
+from .utils import Page
+
+log = logging.getLogger("red.JojoCogs.collectibles")
+log.setLevel(logging.DEBUG)
+_config_structure = {
+    "user": {
+        "collectibles": {},
+        "total_count": 0,
+    },
+    "guild": {
+        "collectibles": {},
+    },
+    "global": {"collectibles": {"Trophy": ["🏆", 300]}},
```

Comment:
```suggestion
    "global": {"collectibles": {"Trophy": ["\N{TROPHY}\N{VARIATION SELECTOR-16}", 300]}},
```
Lets change this to unicode